etherhoused-log

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exec papertrail --min-time "24 hours ago" "program:app/web.1 AND (\"turn_off ${1}\" OR \"turn_on ${1}\")"
```

Comment:
ok :)
Yea whenever I see a webserver exec'ing....

Also you are probably actually watching the *syslog* output, right? That is different that the server logs. The production server could never get the syslog output anyway.